src/PolyhedralGeometry/triangulations.jl

```diff
@@ -0,0 +1,158 @@
+@doc Markdown.doc"""
+    all_triangulations(pts::AnyVecOrMat)
+
+Compute all triangulations on the points given as the rows of `pts`.
+
+The return type is a `Vector{Vector{Vector{Int}}}` where each
+`Vector{Vector{Int}}` encodes a triangulation, in which a `Vector{Int}` encodes
+a simplex as the set of indices of the vertices of the simplex. I.e. the
+`Vector{Int}` `[1,2,4]` corresponds to the simplex that is the convex hull of
+the first, second, and fourth input point.
+
+# Examples
+```jldoctest
+julia> c = cube(2,0,1)
+A polyhedron in ambient dimension 2
+
+julia> V = vertices(c)
+4-element SubObjectIterator{PointVector{Polymake.Rational}}:
+ [0, 0]
+ [1, 0]
+ [0, 1]
+ [1, 1]
+
+julia> all_triangulations(point_matrix(V))
+2-element Vector{Vector{Vector{Int64}}}:
+ [[1, 2, 3], [2, 3, 4]]
+ [[1, 2, 4], [1, 3, 4]]
+```
+"""
+function all_triangulations(pts::AnyVecOrMat)
+    input = homogenize(pts)
```

Comment:
Do we really want to allow a single vector as input?
We could use this instead, which allows passing `vertices(...)` directly:
```suggestion
function all_triangulations(pts::Union{SubObjectIterator{PointVector}, AbstractMatrix, Oscar.MatElem})
    input = homogenized_matrix(pts, 1)
```

We could define some aliases for these unions (for vertexlist, raylist and maybe more?) and unify our signatures a little bit.